docs/common-bugs.rst

```diff
@@ -187,6 +187,14 @@ Failing hardware
 :Solution: It looks like a hardware failure. Please, check your hardware.
 :Example: `rhbz#1685047 <https://bugzilla.redhat.com/show_bug.cgi?id=1685047>`_
 
+LVM on disks with inconsistent sector size
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+:Detection: Error with message mentioning "inconsistent sector size".
```

Comment:
Maybe it could be a sentence? `The storage configuration fails with an error message ...`